client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -90,11 +99,13 @@ class GoogleMyBusinessSelectBusinessType extends Component {
 			connectButton = (
 				<KeyringConnectButton
 					serviceId="google_my_business"
-					onClick={ this.trackCreateYourListingClick }
+					onClick={ this.trackConnectToGoogleMyBusinessClick }
 					onConnect={ this.handleConnect }
+					primary
 				>
-					{ translate( 'Create Your Listing', {
-						comment: 'Call to Action to add a business listing to Google My Business',
+					{ translate( 'Connect to Google My Business', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Connect to your Google My Business account' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/gmb-flows)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).